test/service/test_response.py

```diff
@@ -41,64 +46,59 @@ def test_file_search_response(self):
         appropriate by asserting the apiResponse attribute
         is the same as expected.
         """
-        with open('{}/test1index.json'.format(self.data_folder_filepath)) as json_test:
-            test_json = json.load(json_test)
+        with open(self.hits_input_filename) as hits_input_fp:
+            hits_input = json.load(hits_input_fp)
 
-        with open('{}/filesearch_test1.json'.format(self.data_folder_filepath)) as test1:
-            file_search_test = json.load(test1)
+        with open(self.facet_input_filename) as facet_input_fp:
+            facet_input = json.load(facet_input_fp)
 
-        # This is what will be used as the comparing standard
-        with open('{}/facets_test_input1.json'.format(
-                self.data_folder_filepath)) as facet_input:
-            facet_test = json.load(facet_input)
+        with open(os.path.join(self.data_folder_filepath, "response_pagination_input1.json")) as pagination_input_fp:
```

Comment:
This statement is a bit long. Can you extract the argument to open() into a variable?

Reply:
Here and everywhere else.

Reply:
Addressed, extracted filepaths to setUpClass